org.eclipse.xtext.xtext.ui.examples/projects/org.eclipse.xtext.example.domainmodel/build.properties

```diff
@@ -12,4 +12,7 @@ additional.bundles = org.eclipse.xtext.xbase,\
                      org.eclipse.xtext.xtext.generator,\
                      org.eclipse.emf.codegen.ecore,\
                      org.eclipse.emf.mwe.utils,\
-                     org.eclipse.emf.mwe2.launch
+                     org.eclipse.emf.mwe2.launch,\
+                     org.apache.log4j,\
+                     org.apache.commons.logging,\
+                     com.ibm.icu
```

Comment:
To be consistent with what the Xtext project wizard creates (and the first mwe2 run), this is the correct content:

```
additional.bundles = org.eclipse.xtext.xbase,\
                     org.eclipse.xtext.common.types,\
                     org.eclipse.xtext.xtext.generator,\
                     org.eclipse.emf.codegen.ecore,\
                     org.eclipse.emf.mwe.utils,\
                     org.eclipse.emf.mwe2.launch,\
                     org.eclipse.emf.mwe2.lib,\
                     org.objectweb.asm,\
                     org.apache.commons.logging,\
                     org.apache.log4j,\
                     com.ibm.icu
```

so you still miss some of them